browser/ui/webui/brave_education/education_page_handler.cc

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/webui/brave_education/education_page_handler.h"
+
+#include <array>
+
+#include "base/containers/contains.h"
+#include "brave/browser/ai_chat/ai_chat_utils.h"
+#include "brave/browser/brave_rewards/rewards_service_factory.h"
+#include "brave/browser/brave_wallet/brave_wallet_service_factory.h"
+#include "brave/components/brave_education/common/education_urls.h"
+#include "brave/components/brave_vpn/common/buildflags/buildflags.h"
+#include "brave/components/constants/webui_url_constants.h"
+#include "chrome/browser/profiles/profile.h"
+
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+#include "brave/browser/brave_vpn/vpn_utils.h"
+#endif
+
+namespace brave_education {
+
+namespace {
+
+bool IsCommandSupportedForPageType(mojom::Command command,
+                                   EducationPageType page_type) {
+  switch (page_type) {
+    case EducationPageType::kGettingStarted: {
+      constexpr auto kCommands = std::to_array(
+          {mojom::Command::kOpenRewardsOnboarding,
+           mojom::Command::kOpenWalletOnboarding,
+           mojom::Command::kOpenVPNOnboarding, mojom::Command::kOpenAIChat});
+      return base::Contains(kCommands, command);
```

Comment:
Maybe this could be:

```cxx
static constexpr auto kCommands = base::MakeFixedSet<mojom::Command>({
  mojom::Command::kOpenRewardsOnboarding,
  mojom::Command::kOpenWalletOnboarding,
  mojom::Command::kOpenVPNOnboarding,
  mojom::Command::kOpenAIChat});
return kCommand.contains(command);
```

Reply:
Done.